plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/actions/SelectCommandComboBox.java

```diff
@@ -414,15 +430,15 @@ private void updateMachineActions() {
         machinesListWidget.updatePopup();
 
         if (machinesListWidget.getSelectedName() == null && machinesActions.getChildrenCount() > 0) {
```

Comment:
@vparfonov  I don't think so: `machinesListWidget` and `machineEntryList` are different objects. Anyway we need to ask the author of these rows.